diff -Naur OpenColorIO-1.0.9.orig/CMakeLists.txt OpenColorIO-1.0.9/CMakeLists.txt
--- OpenColorIO-1.0.9.orig/CMakeLists.txt	2013-10-08 17:59:34.000000000 -0500
+++ OpenColorIO-1.0.9/CMakeLists.txt	2013-11-05 21:20:11.824239166 -0600
@@ -190,7 +190,7 @@
     # Set minimum yaml version for non-patched sources.
     set(YAML_VERSION_MIN "0.3.0")
     include(FindPkgConfig)
-    pkg_check_modules(YAML_CPP yaml-cpp)
+    pkg_search_module(YAML_CPP yaml-cpp yaml-cpp03)
     if(YAML_CPP_FOUND)
         if(YAML_CPP_VERSION VERSION_EQUAL ${YAML_VERSION_MIN} OR
            YAML_CPP_VERSION VERSION_GREATER ${YAML_VERSION_MIN})
diff -Naur OpenColorIO-1.0.9.orig/src/core/OCIOYaml.h OpenColorIO-1.0.9/src/core/OCIOYaml.h
--- OpenColorIO-1.0.9.orig/src/core/OCIOYaml.h	2013-10-08 17:59:34.000000000 -0500
+++ OpenColorIO-1.0.9/src/core/OCIOYaml.h	2013-11-05 21:19:23.989303122 -0600
@@ -68,7 +68,7 @@
 
 #endif 
 
-#include <yaml-cpp/yaml.h>
+#include <yaml-cpp03/yaml.h>
 
 #ifndef INCLUDED_OCIO_YAML_H
 #define INCLUDED_OCIO_YAML_H
