diff -Naur imageworks-OpenColorIO-b3cb224.orig/docs/CMakeLists.txt imageworks-OpenColorIO-b3cb224/docs/CMakeLists.txt
--- imageworks-OpenColorIO-b3cb224.orig/docs/CMakeLists.txt	2012-04-17 19:08:16.000000000 -0500
+++ imageworks-OpenColorIO-b3cb224/docs/CMakeLists.txt	2012-04-18 11:18:06.026781578 -0500
@@ -99,7 +99,8 @@
 
 install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/build-html/
         DESTINATION ${CMAKE_INSTALL_PREFIX}/share/doc/OpenColorIO/html
-        PATTERN .git EXCLUDE)
+        PATTERN .* EXCLUDE
+)
 
 find_package(LATEX)
 if(PDFLATEX_COMPILER)
